DOCS/man/options.rst

```diff
@@ -6974,11 +6977,14 @@ them.
 ``--gpu-shader-cache``
     Store and load compiled GLSL shaders in the cache directory (Default:
     ``yes``). Normally, shader compilation is very fast, so this is not usually
-    needed. It mostly matters for anything based on D3D11 (including ANGLE), as
-    well as on some other proprietary drivers. Enabling this can improve startup
-    performance on these platforms.
+    needed. It mostly matters for anything involving GLSL to SPIR-V conversion
+    D3D11 (including ANGLE) or Vulkan, as well as on some other proprietary
+    drivers. Enabling this can improve startup performance on these platforms.
```

Comment:
```
    needed. It mostly matters for anything involving GLSL to SPIR-V conversion,
    that is: D3D11, ANGLE or Vulkan, as well as on some other proprietary
    drivers. Enabling this can improve startup performance on these platforms.
```

"other proprietary drivers" is also very nebulous. do we know what this actually affects?

Reply:
No, it was there before, so I left it as is. I think author was making a statement that mesa works fine with own cache, but if you use anything else it is unknown.

Reply:
NVIDIA on Linux is okay AFAIK as are the mainstream GL drivers on Windows.
who knows :shrug: